opm-core/Backups.rst

```diff
@@ -0,0 +1,22 @@
+Backups
+=======
+
+Usual backup procedures apply. The procedure should include the way to reinstall the extensions on a virgin server.
+
+Restoring logical backups
+-------------------------
+
+In a general way, restoring a backup created with pg_dump should not be a problem as the backup contains the necessary `CREATE EXTENSION` orders. The extension binaries must be present.
```

Comment:
OPM doesn't have any binaries.  If you want to be stick to general recommendation that to apply to OPM, we should mention something like "All extension files (binaries, shared librraries, SQL files...)".

Reply:
Changed